src/components/aboutUs/__tests__/TeamDetail.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { shallow } from "enzyme";
+import TeamDetail from "../TeamDetail";
+
+describe("Test TeamDetail Entry point", () => {
+    it("should be able to render <TeamDetail />", () => {
```

Comment:
Now actual testing in this file. Please add some test cases.

Reply:
Thanks for the suggestions, have been modified and tested in local files.